illumos-utils/src/running_zone.rs

```diff
@@ -797,95 +749,6 @@ impl RunningZone {
         Ok(())
     }
 
-    /// Looks up a running zone based on the `zone_prefix`, if one already exists.
```

Comment:
This function tries to "derive the zone information from the system" if the Sled Agent didn't know about it before.

This is a little hairy for the new `PathInPool` type, because we'd need to re-derive the pool location from the filesystem path. This is possible, but I just opted to not do it, because no one is currently accessing this function.

Instead, given that this function has no callers, I just removed it.